tests/integration/store/rootmulti/rollback_test.go

```diff
@@ -1,85 +0,0 @@
-package rootmulti_test
```

Comment:
nice, i was planning to remove this in the remove testutil/integration PR

Reply:
Yeah store v1 only needs to be tested in 052 branch. This was importing simapp so I had to :D